src/NUnitFramework/tests/Assertions/CollectionAssertTest.cs

```diff
@@ -47,14 +47,14 @@ public class CollectionAssertTest
         [Test()]
         public void ItemsOfType()
         {
-            var collection = new SimpleObjectCollection("x", "y", "z");
+            var collection = new SimpleEnumerable("x", "y", "z");
```

Comment:
I was just trying to match the IEnumerable interface.   But I wouldn't expect this to be merged as is.   Was trying to help communicate the conflicting nature with tests. Fixing one breaks the other when just reordering.